### PR TITLE
Full Implementation of SetCommand

### PR DESCRIPTION
**Pull request by ScarletBlanks**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/StatusTypes.java [hunk 1]
```diff
@@ -19,4 +19,16 @@ public String toString() {
         return this.statusName;
     }
 
+    /**
+     * Checks if the provided status is a valid status.
+     *
+     * @param input The string status to be checked for validity.
+     * @return {@code true} if the status is valid, {@code false} otherwise.
+     */
+    public static boolean isValidStatusType(String input) {
```
Comment (AriellaCallista) [inline src/main/java/seedu/address/model/person/StatusTypes.java:28]: Have you considered using the given `StringUtil.containsWordIgnoreCase` method to check if the given input is a valid status to ignore any case differences?

i.e.

```
public static boolean isValidStatus(String status) {
        List<String> validStatus = Arrays.asList("Preliminary", "Interviewed", "Rejected", "Offered");
        return validStatus.stream()
                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(status, keyword));
}
```

**Review by AriellaCallista — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/StatusTypes.java (hunk 1 above)
Comment (ScarletBlanks) [reply src/main/java/seedu/address/model/person/StatusTypes.java:28]: .toLowerCase is already being used so shouuuuld be fine?

**Review by aritc20 — APPROVED** [review]
LGTM. Could consider standardising the implementation of set status to that of set remark to make testings easier.

**Review by AriellaCallista — APPROVED** [review]
LGTM. Could consider standardising the implementation of set status to that of set remark to make testing easier.

**Pull request merged**